Mol_Analysis.py

```diff
@@ -856,7 +945,9 @@ def process_data(threadName, q):
     parser.add_argument("-begin", type=int, default=0, help="Begin from time X (fs)")
     parser.add_argument("-end", type=int, default=None, help="End at time X (fs)")
     parser.add_argument("-large", action="store_true", help="Can plot 9 plots")
-    parser.add_argument("-splitlen", metavar="slen", default=10, help="Length of timewindow for RMSF calculation (ns). If multiple than separated with comma")
+    parser.add_argument("-splitlen", metavar="slen", default=10, help="Length of timewindow for RMSF and mindist calculation (ns). If multiple than separated with comma")
```

Comment:
I think it makes sense to keep this info just for RMSF (see above). Length of *time window* is a different concept and has a different purpose than what we are doing now

Reply:
you need to correct the text of the helpsstring here